packages/relay/src/lib/services/rateLimiterService/LruRateLimitStore.ts

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
QQ to reviewers: should I keep the store files here or move them to types or another folder such as `types`? They're only related to rateLimiterService so I decided not to move them